geowebcache/rest/src/main/java/org/geowebcache/rest/controller/SeedController.java

```diff
@@ -75,8 +75,13 @@ public ResponseEntity<?> doGet(HttpServletRequest req, @PathVariable String laye
         return seedService.getRunningLayerTasks(req, layer);
     }
 
+    @RequestMapping(value = "/seed/{layer}.xml", method = RequestMethod.GET)
+    public ResponseEntity<?> doGetXml(HttpServletRequest request, @PathVariable String layer) {
+        return formService.handleGet(request, layer);
+    }
```

Comment:
Tested this bit, it does not seem to be necessary:
* formService.handleGet returns HTML outputs anyways
* seedService.getRunningLayerTasks would seem a better fit but it's hard coded to return only JSON (unbelievable, but true...)

However, there is an integration test that would fail without it, ``RestIntegrationTest.testSeedGetLayerXml``... which is not even checking if the output is XML, but would fail if the result is not a 200.
Possible avenues:
* Keep the code as @Judge73 wrote it
* Remove the .xml controller and that test

@tbarsballe git points at you for that test method... althought it might just be a refactor of some pre-existing code. Do you have a preference?

Of course we could also work towards supporting xml output for the seed endpoint, but that seems harder and unrelated to the objective of this PR.

Reply:
Looks like the original changes are coming from https://github.com/GeoWebCache/geowebcache/pull/593 , which is where a number of those endpoints were added in the first place. As such, many of the tests (and for that matter, some of the endpoint implementations) are pretty generic.
In this instance, it looks like we just plugged the REST get directly into the corresponding UI method, which appears to manually generate HTML rather than doing something sensible.

For consistency with the rest of the API (and the documentation for that matter), I'd prefer keeping the XML controller around, even if its still actually returning HTML content for now, and opening a new issue to fix the output to produce XML sometime in the future.
If it was to be removed, the swagger documentation should be updated as well.